src/plat/hifive/overlay-hifive.dts

```diff
@@ -5,8 +5,20 @@
  */
 
 / {
-	chosen {
-		seL4,kernel-devices =
-		    &{/soc/interrupt-controller@c000000};
-	};
+    chosen {
+        seL4,kernel-devices =
+            &{/soc/clint@2000000},
+            &{/soc/interrupt-controller@c000000};
+    };
+
+    /*
+     * According to the SiFive "FU540-C000 Manual" (version v1p4)
+     * the CLINT is mapped from 0x0200_0000 to 0x0200_FFFF.
+     */
+    soc {
+        clint@2000000 {
```

Comment:
Isn't this the same device that's already defined as a child node inside each cpu node? Are there separate copies of the registers for each CPU?

Reply:
> Isn't this the same device that's already defined as a child node inside each cpu node? 

Yes, I thought it would be better to have this change in the overlay rather than the original DTS.

> Are there separate copies of the registers for each CPU?

I don't think the registers are banked if that's what you're asking, the memory map [here](https://sifive.cdn.prismic.io/sifive/da6d8344-efb9-4732-875c-4926eb5eff77_u54mc_core_complex_manual_21G2.pdf#%5B%7B%22num%22%3A412%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C0%2C678%2C0%5D) shows separate registers for each HART. However, it looks like there's only one `mtime` register for the whole system.